dapps/profile.vue/src/components/did/Delegates.vue

```diff
@@ -0,0 +1,137 @@
+  the following URL: https://evan.network/license/
+*/
+
+<template>
+  <div class="content-card">
+    <div class="card-heading mb-3 d-flex flex-row justify-content-between align-items-center">
+      <h2>
+        <i class="mr-1 mdi mdi-earth" />
+        {{ '_profile.did.delegates-title' | translate }}
+      </h2>
+      <evan-button
+        v-if="!isEditMode"
+        class="btn-sm"
+        @click="onEditStart"
+      >
+        {{ '_evan.edit' | translate }}
+      </evan-button>
+    </div>
+    <p>{{ '_profile.did.delegates-desc' | translate }}</p>
+
+    <evan-table
+      class="simple"
+      :fields="columns"
+      :items="delegates"
+    >
+      <template v-slot:empty>
+        {{ '_profile.did.empty-delegates' }}
+      </template>
+
+      <template
+        v-if="isEditMode"
+        v-slot:cell(label)="data"
+      >
+        <evan-form-control-input
+          class="clean-input"
+          :value="data.item.label"
+        />
+      </template>
+
+      <template
+        v-if="isEditMode"
+        v-slot:cell(url)="data"
+      >
+        <evan-form-control-input
+          class="clean-input"
+          :value="data.item.url"
+        />
+      </template>
+
+      <template v-slot:cell(action)="data">
+        <evan-button
+          v-show="isEditMode"
+          type="icon-secondary"
+          icon="mdi mdi-trash-can-outline"
+          @click="deleteDelegate(data.index)"
+        />
+      </template>
+
+      <template
+        v-if="isEditMode"
+        v-slot:bottom-row
+      >
+        <b-td colspan="3">
+          <!-- $props and $attr binding with evan-form-control-v-select isn't ideal.
+          Using "native" v-select instead -->
+          <evan-v-select
+            class="m-0"
+            :value="null"
+            :options="contacts"
+            :placeholder="'_profile.did.choose-contact' | translate"
+            :clear-search-on-select="true"
+            @input="onSelectContact"
+          />
+        </b-td>
+      </template>
+    </evan-table>
+
+    <div
+      v-if="isEditMode"
+      class="d-flex mt-3 align-items-center justify-content-between"
+    >
+      <div class="d-flex align-items-center">
+        <i class="mdi mdi-information-outline mr-2" />
+        <span>{{ '_evan.transaction_costs_hint' | translate }}</span>
+      </div>
+      <div>
+        <evan-button @click="onEditCancel">
+          {{ '_evan.cancel' | translate }}
+        </evan-button>
+        <evan-button
+          class="ml-1"
+          type="primary"
+          :disabled="!hasChanges"
+          @click="saveDelegates"
+        >
+          {{ '_evan.save' | translate }}
+        </evan-button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Component from './Delegates';
+
+export default Component;
+</script>
+
+<style lang="scss" scoped>
+.content-card {
+  background: white;
+  border-radius: 4px;
+  width: 560px;
+  padding: 24px 24px;
+
+  .card-heading h2 {
```

Comment:
isn't it the same as hour headline styles?

Reply:
True, this can be removed :)